group/urls.py

```diff
@@ -3,5 +3,8 @@
 from . import views
 
 urlpatterns = [
-    path('', views.create_group)
+    path('', views.create_group),
+    path('<int:group_id>/user/', views.add_users),
+    path('<int:group_id>/delete/', views.delete_group),
+    path('<int:group_id>/remove/<int:user_id>', views.remove_user),
```

Comment:
Why are you passing both a group and user ID in the same path? There should be a better way to handle this. You can send the group information in the request body for example, instead of having it as part of the url